assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -374,16 +374,13 @@ export default function SetupBanner( { onSubmitSuccess } ) {
 				</SpinnerButton>
 			}
 			footer={
-				<Fragment>
-					{ footerMessages.map( ( message, index ) => (
-						<p
-							className="googlesitekit-ga4-setup-banner__footer-text"
-							key={ index }
-						>
-							{ message }
-						</p>
-					) ) }
-				</Fragment>
+				!! footerMessages.length && (
+					<ul className="googlesitekit-ga4-setup-banner__footer-text-list">
+						{ footerMessages.map( ( message, index ) => (
+							<li key={ index }>{ message }</li>
+						) ) }
```

Comment:
Using the index as a React key is usually discouraged, because theoretically the order could change and it's less performant.

That's a mild concern here, but there's no reason not to use a string (eg. the message text itself) as the key here, so let's do that.

```suggestion
						{ footerMessages.map( ( message ) => (
							<li key={ message }>{ message }</li>
						) ) }
```